root/defaults/default

```diff
@@ -107,6 +110,37 @@ server {
 #	}
 #}
 
+# sample reverse proxy config for "heimdall" via subdomain, with Authelia
+# Authelia container has to be running, the /config/nginx/authelia.conf file should be filled with relevant info and authelia.subdomain.conf proxy conf should be enabled
```

Comment:
```suggestion
# Authelia container has to be running, the /config/nginx/authelia-server.conf file should be filled with relevant info and authelia.subdomain.conf proxy conf should be enabled
```

Reply:
Ah, you're right, I forgot to update that. Thanks, I'll fix it

Reply:
updated in https://github.com/linuxserver/docker-letsencrypt/pull/460/commits/f75ca59cfa2c9d22dab5746910df47cd9c4ea8c1